packages/space-opera/src/components/camera_settings/camera_settings.ts

```diff
@@ -109,7 +113,7 @@ export class CameraTargetInput extends ConnectedLitElement {
   @internalProperty() target?: Vector3D;
 
   stateChanged(state: State) {
-    this.target = state.camera.target ?? state.initialCamera.target;
+    this.target = getCamera(state).target ?? getInitialCamera(state).target;
```

Comment:
getCameraTarget could be a selector that I made, but I don't think it is necessary. This is a more of a design decision.

Reply:
yeah